mod-audit-server/src/main/java/org/folio/dao/acquisition/impl/OrderEventsDaoImpl.java

```diff
@@ -85,11 +90,11 @@ private void makeSaveCall(Promise<RowSet<Row>> promise, String query, OrderAudit
         orderAuditEvent.getAction(),
         orderAuditEvent.getOrderId(),
         orderAuditEvent.getUserId(),
-        LocalDateTime.ofInstant(orderAuditEvent.getEventDate().toInstant(), ZoneOffset.UTC),
-        LocalDateTime.ofInstant(orderAuditEvent.getActionDate().toInstant(), ZoneOffset.UTC),
+        LocalDateTime.ofInstant(orderAuditEvent.getEventDate().toInstant(), ZoneId.systemDefault()),
+        LocalDateTime.ofInstant(orderAuditEvent.getActionDate().toInstant(), ZoneId.systemDefault()),
```

Comment:
Just for may knowledge why we need to change ZoneOffset here?

Reply:
first orderAuditEvent.getActionDate().toInstant() this part converting date to seconds 
second LocalDateTime.ofInstant(orderAuditEvent.getActionDate().toInstant(), ZoneId.systemDefault()) localdatetime creating date time from second based on system zoneId